packages/landing/src/pages/landing-content.mdx

```diff
@@ -10,15 +10,21 @@ leveraging the language to create expressive & dynamic experiences.
 ```jsx
 /** @jsxImportSource @compiled/react */
 
-export const LargeText = ({ color, children }) => (
-  <span
-    css={{
-      color: color,
-      fontSize: 48,
-    }}>
-    {children}
-  </span>
-);
+import { css } from '@compiled/react';
+
+export const LargeText = ({ color, children }) => {
```

Comment:
I was debating to myself whether to use a `style` attribute instead of a `css` function call for the `color: color` part or not

Ultimately I decided to use the `css` function call despite it not following the UI Styling Standard, as

1. the next section on the page shows how dynamic styling like `color: color` gets processed in Compiled
2. there are no eslint rules within Compiled that forbid `css({ color: color })` in this way

Reply:
I didn't really know this was even allowed with Compiled's future state 😄—Atlassian notwithstanding—but I have no qualms with documenting Compiled's current feature set like this!

Added a comment on the lines for an alternative suggestion.

Reply:
Yeah keep in mind that we're documenting Compiled, not the UI styling standard. We don't want to be removing stuff from the docs just because we don't have it in our happy path.